.drone.yml

```
@@ -37,12 +37,12 @@ pipeline:
       branch: master
 
   run-python-tests:
-    image: python:3
+    image: python:3.8
     commands:
       - cd python
-      - pip install poetry
-      - poetry install 
-      - "TEST=1 ./test_and_coverage.sh"
+      - curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
+      - $HOME/.poetry/bin/poetry install 
+      - $HOME/.poetry/bin/poetry run "TEST=1 ./test_and_coverage.sh"
```

it looks like passing the whole command sequence/argument as a single string is making something further down the line treat it as a whole file with a space in its name. is it possible to move the TEST=1 to the environment section? i believe vars in the environment section will be exported so should be visible by poetry and child processes, unless im mistaken glubglub.

```suggestion
      - $HOME/.poetry/bin/poetry run ./test_and_coverage.sh
    environment:
      - TEST=1
```